### PR TITLE
invite member modal fits on screen for small screen

### PR DESCRIPTION
**Pull request by salmanmanekia**
#### Summary

The PR fixes the squashed invite member modal which happens on smaller screens.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-44799

#### Screenshots

|  before  |  after  |
|----|----|
| Screenshot 2022-09-05 at 10 58 12 | Screenshot 2022-09-05 at 10 54 34 |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/invitation_modal/invitation_modal.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .InvitationModal {
     .modal-dialog {
-        position: absolute;
+        @media screen and (min-width: 640px) {
+            position: absolute;
+        }
+
```
Comment (julmondragon) [inline components/invitation_modal/invitation_modal.scss:6]: nit: For a screen width between 768 and 640px the dialog still shown a bit reduced:

https://user-images.githubusercontent.com/79058848/188924087-9f3a40a9-ca55-43e8-876c-954d1a31bc06.mp4

**Review by julmondragon — APPROVED** [review]
LGTM ! Thanks @salmanmanekia

**Review by JulienTant — APPROVED** [review]
Nothing to add to @julmondragon review! Nice job @salmanmanekia

**Comment by annelieseklein** [comment]
@salmanmanekia looks great! Can we center the modal vertically please?

**Review by annelieseklein — APPROVED** [review]

**Comment by salmanmanekia** [comment]
Thank you team. I have made the suggested changes in the PR.

**Pull request merged**